Kyrychenko/quizEntity/src/main/java/ui/QuizServlet.java

```diff
@@ -1,31 +1,72 @@
 package ui;
 
 import model.Quiz;
+import model.User;
 import service.QuizService;
+import service.UserService;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.Objects;
 
 
 public class QuizServlet extends HttpServlet {
+    private static final String PARAM_NAME_USERNAME = "name";
+    private static final String PARAM_NAME_LOGIN = "login";
+    private static final String PARAM_NAME_PASSWORD = "password";
+
+    private static final String LOGIN_PAGE = "/login.jsp";
+    private static final String REGISTRATION_PAGE = "/registration.jsp";
+    private static final String ADD_QUIZ_PAGE = "/view/inputQuiz.jsp";
+    private static final String VIEW_QUIZ_PAGE = "/view/outputQuiz.jsp";
+
+    private static final String ERROR_EMPTY_LOGIN_PASS_MESSAGE = "One of the entered values are empty";
+    private static final String ERROR_LOGIN_PASS_MESSAGE = "Incorrect login or password";
+    private static final String ERROR_LOGIN_EXIST_MESSAGE = "User with such login already exist";
+
     private static QuizService quizService = QuizService.getInstance();
+    private static UserService userService = UserService.getInstance();
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
         String action = req.getParameter("action");
 
+        String login = req.getParameter(PARAM_NAME_LOGIN);
+        String password = req.getParameter(PARAM_NAME_PASSWORD);
+
+        HttpSession session = req.getSession();
+
         switch (action == null ? "add" : action) {
+            case "login":
+                if (isEmptyParameters(login, password) || isNullParameters(login, password)) {
+                    forwardWithErrorMessage(req, resp, LOGIN_PAGE,  ERROR_EMPTY_LOGIN_PASS_MESSAGE);
```

Comment:
better separate this logic into another servlet. Switch, if, else if, absolutely unreadable.